utils/build/virtual_machine/provisions/local-auto-inject-install-script/auto-inject_apm_libraries_from_binaries.yml

```diff
@@ -0,0 +1,12 @@
+#Execute script installation for lib-injection packages using install script
+- os_type: linux
+  copy_files:
+      - name: copy-binaries
+        local_path: binaries
+      - name: copy-start-agent-script
+        local_path: utils/build/virtual_machine/microvm/microvm_agent_restart_retry.sh
+  remote-command: |
+   mkdir /shared_volume/datadog-apm-library
+   find /shared_volume -name "datadog-apm-library*.tar" -exec sh -c 'tar xvf {} -C /shared_volume/datadog-apm-library' \;
+   sudo datadog-installer install file:///shared_volume/datadog-apm-library
+   bash /shared_volume/microvm_agent_restart_retry.sh
```

Comment:
Why are we doing this? Can't we authenticate to either registry.ddbuild.io or the QA registry?
The file install hasn't been tested or isn't maintained for anything else than the installer's first install, I'm a bit worried about stability

Reply:
This is only for local testing with krunvm. We are using a tar file located on the binaries folder. This tar file, should be download manually from the pipeline (package-oci job)

Reply:
It'd be much simpler to just specify the pipeline ID than to go fetch the dependencies in the pipeline. Is there no way to use registry.ddbuild.io or the QA registry in KrunVMs? If the VMs inherit the host network they may be able to target registries behind AppGate no?

Reply:
We can point this out as an improvement. Currently Krunvm are using them in the same way, downloading the artifact from CI and copying it to the "binaries" folder.